apps/homescreen/test/marionette/change_wallpaper_test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
```

Comment:
Before this PR is merged, this file should not be in it. You need to either create a new commit which deletes the file or to erase https://github.com/mozilla-b2g/gaia/commit/c9dd786d9e79da721206ae4b5a8542ad6a66f0cd from your branch. 

In either way, it'll be the same as long as you squash the commits into 1.